func/definitions.h

```diff
@@ -4,6 +4,7 @@
 #define FILE_CATEGORY_DELIMITER "."
 #define MAX_FILE_LINE_LENGTH 128
 #define MAX_RECIPE_NAME 128
+#define MAX_INGREDIENT_CATEGORY 128
```

Comment:
```suggestion
#define MAX_INGREDIENT_CATEGORY_NAME 128
```
Consider calling this something with name, to make it easier to understand what it does :)